package maidsafe.vault.protobuf;
option optimize_for = LITE_RUNTIME;

message Sync {
  optional int32 account_type = 1;
  required bytes account_name = 2;
  required bytes serialised_unresolved_entries = 3;
}
